samcli/local/docker/manager.py

```diff
@@ -168,16 +168,16 @@ def pull_image(self, image_name, tag=None, stream=None):
                 raise DockerImagePullFailedException(str(ex)) from ex
 
             # io streams, especially StringIO, work only with unicode strings
-            stream_writer.write("\nFetching {}:{} Docker container image...".format(image_name, tag))
+            stream_writer.write_str("\nFetching {}:{} Docker container image...".format(image_name, tag))
 
             # Each line contains information on progress of the pull. Each line is a JSON string
             for _ in result_itr:
                 # For every line, print a dot to show progress
-                stream_writer.write(".")
-                stream_writer.flush()
+                stream_writer.write_str(".")
+                # stream_writer.flush()
```

Comment:
nit: left over from debugging I assume

Reply:
Ya. I don't think flush is required here but will leave it. Shouldn't be hurting anything.